humman_mogen/README.md

```diff
@@ -0,0 +1,131 @@
+# HuMMan Release v1.0: Reconstruction Subset
+
+HuMMan v1.0: Mogen Subset consists of 179 subjects, 6264 sequences. 
+SMPL parameters, stage divisions, overall & bodypart annotations and video visualizations are provided. 
+
+### Downloads
```

Comment:
Files can be downloaded from [this link](https://openxlab.org.cn/datasets/OpenXDLab/HuMMan/tree/main/humman_release_v1.0_mogen)